paper.md

```diff
@@ -74,37 +81,56 @@ of the following classes:
 implementation using the fast Fourier transform [@frigo2005design].
 
 In addition to basic SBP derivative operators, SummationByPartsOperators.jl
-contains a number of artificial dissipation and filtering operators, such as
+contains a number of related operators, such as
 
 - SBP artificial dissipation operators
 - spectral viscosity operators for Fourier methods
 - modal filter operators for Fourier methods and Legendre pseudospectral methods
 
+Using Julia's rich type system, all of these operators are implemented as their
+own types. This enables several optimizations such as a memory requirement
+independent of the number of grid points. In contrast, implementations based
+on sparse/banded matrices have a memory requirement growing linearly with the
+number of grid points. In addition, the structure of the operators can be taken
+into account for operator-vector multiplications, usually resulting in speed-ups
+of an order of magnitude or more on consumer hardware. For example, the application
+of the sixth-order (in the interior) finite difference SBP operator on a grid
+with 1000 nodes takes roughly 330 ns on a consumer CPU from 2017 (Intel® Core™ i7-8700K)
+using version v0.5.5 of SummationByPartsOperators.jl. In contrast, the same
+operation takes roughly 3.9 microseconds using a sparse matrix format used in
+other implementations of this operator [@almquist2017optimized].
+
 Following good software development practices, SummationByPartsOperators.jl
 makes use of continuous integration and automated tests required before merging
-pull requests. Documentation is provided both in form of docstrings, a general
-introduction, and tutorials.
+pull requests. Documentation is provided in form of docstrings, a general
+introduction, and tutorials. In addition, SummationByPartsOperators.jl is a
+registered Julia package and can be installed using the built-in package manager,
+handling dependencies and version requirements automatically.
 
 
 # Related research and software
 
 There are of course many open-source software packages providing discretizations
 of differential equations. However, many of them focus on a single class of
-numerical methods, e.g.,
-
-- finite difference methods ([DiffEqOperators.jl](https://github.com/SciML/DiffEqOperators.jl), a part of DifferentialEquations.jl [@rackauckas2017differentialequations])
-- finite volume methods [@ramadhan2020oceananigans;@xiao2021kinetic]
-- spectral methods [@olver2014practical;@constantinou2021fourierflows]
-- finite element methods [@badia2020gridap]
-- discontinuous spectral element methods [@schlottkelakemper2021purely;@schlottkelakemper2020trixi]
+numerical methods or a specific application, e.g.,
+
+- finite difference methods ([DiffEqOperators.jl](https://github.com/SciML/DiffEqOperators.jl),
+  a part of DifferentialEquations.jl [@rackauckas2017differentialequations])
+- finite volume methods (Oceananigans.jl [@ramadhan2020oceananigans],
```

Comment:
Here we still have a slight asymmetry. the first package has its name linked to the repository. Would it maybe be reasonable to add also links to the repositories for the other packages as well?

Reply:
I did that in 31cda8d. I hesitated to do so since URLs of software repositories can likely change in the future. However, linking the repo seemed to be the only option to cite DiffEqOperators.jl correctly since there is no paper on it, in contrast to the other software packages.

Reply:
For all that have a zenodo DOI you could also link that one. I was just surprised by the asymmetry. I think it is good to have a link for each of the packages, makes them easier accessible. And I think the URLs should not change that much (and if a Repo is moved the old URL works for quite a while on GitHub – see for example https://github.com/JuliaNLSolvers/Manifolds.jl – and we moved the repository a long time ago).

Reply:
I did that in d2962f3 for all packages that provide a Zenodo DOI badge in their README.

Reply:
I personally prefer the GitHub URLs, but your solution is also fine.

Reply:
I reverted that back to GitHub URLs in b38a64a. Please resolve this conversation if that state is okay for you.

Reply:
I can not resolve this (I do not know why, maybe due to the force pushs?) but sure this is ok now.